Sources/KlaviyoSwift/InternalAPIModels.swift

```diff
@@ -121,8 +123,23 @@ extension KlaviyoAPI.KlaviyoRequest {
                     let uniqueId: String
                     init(attributes: KlaviyoSwift.Event,
                          anonymousId: String? = nil) {
+                        let context = KlaviyoAPI.KlaviyoRequest._appContextInfo
```

Comment:
@ndurell FYI I moved where I appended the metadata to the event from the Event model to this "Internal" model, that seemed more appropriate.